.github/workflows/test.yml

```diff
@@ -0,0 +1,55 @@
+name: Test
+on: [push, pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    strategy:
+      matrix:
+        node: [ 16, 18 ]
```

Comment:
I've left Node v14 out of this, since until #63 is merged, it currently doesn't work.